FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth/DemoTasks/MutualAuthMQTTExample.c

```diff
@@ -572,6 +586,14 @@ static void prvCreateMQTTConnectionWithBroker( MQTTContext_t * pxMQTTContext,
      * the keep-alive period, the MQTT library will send PINGREQ packets. */
     xConnectInfo.keepAliveSeconds = mqttexampleKEEP_ALIVE_TIMEOUT_SECONDS;
 
+    /* Use the username and password for authentication if they are defined. */
+    #ifdef democonfigCLIENT_USERNAME
+        xConnectInfo.pUserName = democonfigCLIENT_USERNAME;
+        xConnectInfo.userNameLength = ( uint16_t ) strlen( democonfigCLIENT_USERNAME );
+        xConnectInfo.pPassword = democonfigCLIENT_PASSWORD;
+        xConnectInfo.passwordLength = ( uint16_t ) strlen( democonfigCLIENT_PASSWORD );
```

Comment:
What about ALPN settings needed for custom auth?

Reply:
Product 3 demos are for connecting to Mosquitto rather than AWS IoT. Since ALPN and SNI are not required for username/password, there is no need to set them.

Reply:
I thought the plan was to support both AWS IoT MQTT broker and Mosquitto local broker depending on the user config. Has that plan changed?

Reply:
It would be good to have a user-config like `USE_AWS_IOT_CORE_BROKER` based on which the ALPN settings can be set for custom auth

Reply:
Added those options